pkg/sdn/apis/network/v1/types.go

```diff
@@ -20,6 +20,8 @@ type ClusterNetwork struct {
 
 	// Network is a CIDR string specifying the global overlay network's L3 space
 	Network string `json:"network" protobuf:"bytes,2,opt,name=network"`
+	// PLACEHOLDER
```

Comment:
HostSubnetLength is part of ClusterNEtworkEntry now. 

The size of each HostSubnetLength is compared to its ClusterCIDR in
pkg/sdn/pluginapis/network/validation/validation.go in ValidateClusterNetwork()